corpora/csip/metadata/mets-root/CSIP3/testCase.xml

```diff
@@ -1,23 +1,87 @@
+    - MUST exist;
+    - MUST have a string value declaring the content category of the package/representation;
+    - MUST NOT have a value from the VocabularyContentCategory because these vocabulary values MUST be used for the attribute mets/@TYPE.
+    If mets/@csip:OTHERTYPE exists then mets/@TYPE MUST be "OTHER".
+  </description>
   <dependencies>
+    <dependency requirementId="CSIP2" URL="https://earkcsip.dilcis.eu/#CSIP2">
+      The value of the other attribute determines the existence of this attribute.
+    </dependency>
+    <dependency requirementId="VocabularyContentCategory" URL="https://earkcsip.dilcis.eu/schema/CSIPVocabularyContentCategory.xml">
+      The value of the attribute MUST NOT be from the vocabulary.
+    </dependency>
   </dependencies>
-  <!-- A list of the validation rules derived from the test case -->
   <rules>
+    <!--  The following rules are included in the test case for CSIP2 and not repeated here:
+          - "If mets/@TYPE = "OTHER" then mets/@csip:OTHERTYPE MUST exist."
+          - "If mets/@TYPE = "OTHER" then mets/@csip:OTHERTYPE MUST exist and have a value."-->
+    <!-- This rule is implicit in the specification and is important for CSIP2 and CSIP3 to work as intended. -->
+    <rule id="1">
+      <description>If mets/@TYPE = "OTHER" then mets/@csip:OTHERTYPE MUST NOT have a value from the VocabularyContentCategory because these vocabulary values MUST be used for the attribute mets/@TYPE.</description>
+      <error level="ERROR">
+        <message>
+          mets/@csip:OTHERTYPE attribute has a value that MUST instead be used for mets/@TYPE. MUST be: mets/@csip:OTHERTYPE MUST describe a content category that falls outside of the VocabularyContentCategory vocabulary.
+        </message>
+      </error>
+      <corpusPackages>
+        <!-- Ideally, there should be an invalid IP for each specific vocabulary
+          value, as this is the only certain way to test that the validator
+          checks for all of them. This should be automated as
+          VocabularyContentCategory currently allows 25 different values. -->
+        <package name="mets-xml_mets_TYPE_OTHER_and_csip-OTHERTYPE_vocabulary_1" isValid="FALSE">
+          <path></path>
+          <description>
+            METS.xml file has mets/@TYPE = "OTHER", mets/@csip:OTHERTYPE = "Textual works - Print".
+          </description>
+        </package>
+        <package name="minimal_IP_with_1_rep_mets_TYPE_OTHER" isValid="TRUE">
+          <path></path>
+          <description>
+            Minimal IP that matches all MUST requirements, mets/@TYPE = "OTHER", mets/@csip:OTHERTYPE = "Textual works - Manuscripts".
+          </description>
+        </package>
+      </corpusPackages>
+    </rule>
+    <!-- This rule is implicit in the specification and is important for CSIP2 and CSIP3 to work as intended. -->
+    <rule id="2">
+      <description>If mets/@csip:OTHERTYPE exists then mets/@TYPE MUST be "OTHER".</description>
```

Comment:
or also MIXED?

Reply:
I see the concern but I don't have enough experience to know if it's an issue in real life. If it is, we should propose a change to the A2 team. At the moment the CSIP5 description mentions only "OTHER," so I left it unchanged.

Reply:
I agree - the feedback to the A2 must be that we need to know how to handle "MIXED"